WTWSMS/history/titles/d_odoacer.txt

```diff
@@ -0,0 +1,4 @@
+476.1.1={
+	holder = 55 # Odoacer
+	set_global_flag = italy_bureaucratic
+}
```

Comment:
If Odoacer does not hold Italy at start I don't think he should be Bureaucratic, in my view he would adopt this government after winning. In the pull request he is pre-feudal, he should be Sub-Roman.

Reply:
> His title is also kept after the victory, it should disappear at that point.

It was supposed to disappear, lemme check.

Reply:
I looked in history, I didn't try winning

Reply:
I think I've fixed the aforementioned issues.